common-lib/src/cmr/common/test/runners/ltest.clj

```diff
@@ -0,0 +1,38 @@
+(ns cmr.common.test.runners.ltest
+  "This test runner is a CMR adaptor for the ltest Clojure test runner."
+  (:require
+   [clojure.stacktrace :as stack]
+   [clojure.string :as string]
+   [clojure.test :as test]
+   [cmr.common.test.runners.util :as util]
+   [ltest.core :as ltest]
+   [potemkin :refer [import-vars]]))
+
+(import-vars
```

Comment:
It also has the one really annoying side-effect of breaking jump-to-definition. Not a big deal for a utility namespace like this, but really annoying when you are trying to navigate the code for an API to see how things are implemented.